plugins/offboard/offboard.h

```diff
@@ -22,31 +37,83 @@ class Offboard
         UNKNOWN
     };
 
+    /**
+     * Returns human-readable English string for Offboard::Result.
+     *
+     * @param result result enum
+     */
     static const char *result_str(Result);
 
+    /**
+     * Callback type for offboard requests.
+     */
     typedef std::function<void(Result)> result_callback_t;
 
+    /**
+     * Type for Velocity commands in NED (North East Down) coordinates and yaw.
+     */
     struct VelocityNEDYaw {
-        float north_m_s;
-        float east_m_s;
-        float down_m_s;
-        float yaw_deg;
+        float north_m_s; /**< Velocity North in meters/second */
+        float east_m_s; /**< Velocity East in meters/second */
+        float down_m_s; /**< Velocity Down in meters/second */
+        float yaw_deg; /**< Yaw in degrees (0 North, positive is clock-wise looking from above. */
     };
 
+    /**
+     * Type for velocity commands in body coordinates (forward, right, down and yaw angular rate).
+     */
     struct VelocityBodyYawspeed {
-        float forward_m_s;
-        float right_m_s;
-        float down_m_s;
-        float yawspeed_deg_s;
+        float forward_m_s; /**< Velocity forward in meters/second */
+        float right_m_s; /**< Velocity right in meters/second */
+        float down_m_s; /**< Velocity down in meters/second */
+        float yawspeed_deg_s; /**< Yaw angular rate in degrees/second (positive for clock-wise
+                                   looking from above */
     };
 
+    /**
+     * Starts offboard control (synchronous).
+     *
+     * Attention: this is work in progress, use with caution!
+     *
+     * @return result of request
+     */
     Offboard::Result start() const;
+
+    /**
+     * Stops offboard control (synchronous).
+     *
+     * @return result of request
+     */
     Offboard::Result stop() const;
 
+    /**
+     * Starts offboard control (asynchronous).
+     *
+     * Attention: this is work in progress, use with caution!
+     *
+     * @param result to receive request result
```

Comment:
perhaps `@param result Callback to receive request result.`

Reply:
ok.